codebases/index.md

```diff
@@ -0,0 +1,31 @@
+
+# Where we work: our codebases
+
+The Foundation for Public Code helps to effectively scale codebases. We do this by supporting ambitious communities with [codebase stewardship](../codebase-stewardship/index.html).
+
+Codebases are organized by [codebase maturity](https://about.publiccode.net/activities/codebase-stewardship/lifecycle.html).
+
+## Assessment
+
+In this phase, codebase communities and the Foundation for Public Code decide if:
+
+* the codebase is a good fit for Foundation stewardship
+* Foundation stewardship will meet the community's goals and ambitions
+
+These codebases are currently in public assessment:
+
+* [Signalen](https://github.com/Amsterdam/signals): receiving, automatically classifying and prioritising residents’ nuisance reports
```

Comment:
I am not sure, but should we perhaps link this to signalen.org instead?

Reply:
If we wish to highlight the strong role of Amsterdam, perhaps link to "Signalen" at signalen.org with text like "based on SIA" linked to Amsterdam/signals?

Reply:
Good spot, thank you!